packages/eslint-plugin-pf-codemods/lib/rules/v5/label-remove-isTruncated.js

```diff
@@ -0,0 +1,11 @@
+const { renameProp } = require('../../helpers');
+
+  // https://github.com/patternfly/patternfly-react/pull/8771
+  module.exports = {
+    meta: { fixable: 'code' },
+    create: renameProp(
+      'Label',
+      {'isTruncated': ''},
+      node =>  `isTruncated prop has been removed for ${node.name.name}. This is now the default behavior.`
```

Comment:
Could also mention the `textMaxWidth` prop as a means to customize when truncation occurs.

Reply:
i thought about this, and included it in the readme. I'm not sure it's worth putting here as if they currently have it implemented with `isTruncated` it's going to be working as they want in that implementation (maybe a wrapper of some sort). I don't know that I would expect products to refactor at this point.

But... I suppose it wouldn't hurt to mention.

Reply:
Yeah I don't feel super strong about it, figured I'd mention it in case, though.